src/test/java/org/sagebionetworks/bridge/spring/controllers/StudyParticipantControllerTest.java

```diff
@@ -221,6 +228,67 @@ public void verifyAnnotations() throws Exception {
         assertDelete(StudyParticipantController.class, "deleteActivityEvent");
     }
     
+    @Test
+    public void getParticipantRoster() throws Exception {
+        RequestContext.set(new RequestContext.Builder()
+                .withOrgSponsoredStudies(ImmutableSet.of(TEST_STUDY_ID)).build());
+        
+        session.setParticipant(new StudyParticipant.Builder()
+                .withId(TEST_USER_ID).withEmail(EMAIL).withEmailVerified(true).build());
+        
+        doReturn(session).when(controller).getAuthenticatedSession(STUDY_COORDINATOR);
+        
+        mockRequestBody(mockRequest, 
+                createJson("{'studyId':'"+TEST_STUDY_ID+"', 'password': '"+PASSWORD+"'}"));
```

Comment:
This should be overridden by the studyId in the route. that's what we're checking for access.

Reply:
Changed this for study coordinators (for researchers they can request any study, or no study)